2025-08-14 15:14:18,048 INFO: GitLab Analytics Hub startup [in D:\Documents\Code\gitlab-automation-suite\app\__init__.py:32]
2025-08-14 15:14:18,049 INFO: --- PATH DEBUGGING --- [in D:\Documents\Code\gitlab-automation-suite\app\__init__.py:35]
2025-08-14 15:14:18,049 INFO: App Root Path: D:\Documents\Code\gitlab-automation-suite\app [in D:\Documents\Code\gitlab-automation-suite\app\__init__.py:36]
2025-08-14 15:14:18,050 INFO: App Template Folder: templates [in D:\Documents\Code\gitlab-automation-suite\app\__init__.py:37]
2025-08-14 15:14:18,050 INFO: Full Template Path: D:\Documents\Code\gitlab-automation-suite\app\templates [in D:\Documents\Code\gitlab-automation-suite\app\__init__.py:38]
2025-08-14 15:14:18,051 INFO: --- END PATH DEBUGGING --- [in D:\Documents\Code\gitlab-automation-suite\app\__init__.py:39]
2025-08-14 15:14:20,493 INFO: --- BLUEPRINT DEBUGGING --- [in D:\Documents\Code\gitlab-automation-suite\app\__init__.py:46]
2025-08-14 15:14:20,494 INFO: Registered Blueprint: 'main' [in D:\Documents\Code\gitlab-automation-suite\app\__init__.py:48]
2025-08-14 15:14:20,494 INFO:   - Blueprint Template Folder: ../templates [in D:\Documents\Code\gitlab-automation-suite\app\__init__.py:49]
2025-08-14 15:14:20,494 INFO: --- END BLUEPRINT DEBUGGING --- [in D:\Documents\Code\gitlab-automation-suite\app\__init__.py:50]
2025-08-14 15:19:24,937 INFO: Successfully connected to GitLab at https://emu.hinagro.com [in D:\Documents\Code\gitlab-automation-suite\app\main\services.py:27]
2025-08-14 15:19:32,208 INFO: Successfully connected to GitLab at https://emu.hinagro.com [in D:\Documents\Code\gitlab-automation-suite\app\main\services.py:27]
2025-08-14 15:19:45,847 INFO: Successfully connected to GitLab at https://emu.hinagro.com [in D:\Documents\Code\gitlab-automation-suite\app\main\services.py:27]
2025-08-14 15:19:54,229 INFO: Successfully connected to GitLab at https://emu.hinagro.com [in D:\Documents\Code\gitlab-automation-suite\app\main\services.py:27]
2025-08-14 15:19:54,230 INFO: Generating Issue TAT Trend Report for 3 months. [in D:\Documents\Code\gitlab-automation-suite\app\main\logic.py:211]
2025-08-14 15:20:09,597 INFO: Fetched ALL 760 issues with filters: {'state': 'closed', 'created_after': '2025-05-14', 'created_before': '2025-08-14', 'all': True, 'include_subgroups': True} [in D:\Documents\Code\gitlab-automation-suite\app\main\services.py:279]
2025-08-14 15:21:48,917 INFO: Successfully connected to GitLab at https://emu.hinagro.com [in D:\Documents\Code\gitlab-automation-suite\app\main\services.py:27]
2025-08-14 15:22:01,530 INFO: Successfully connected to GitLab at https://emu.hinagro.com [in D:\Documents\Code\gitlab-automation-suite\app\main\services.py:27]
2025-08-14 15:22:29,057 INFO: Successfully connected to GitLab at https://emu.hinagro.com [in D:\Documents\Code\gitlab-automation-suite\app\main\services.py:27]
2025-08-14 15:22:39,970 INFO: Successfully connected to GitLab at https://emu.hinagro.com [in D:\Documents\Code\gitlab-automation-suite\app\main\services.py:27]
2025-08-14 15:22:56,393 INFO: Successfully connected to GitLab at https://emu.hinagro.com [in D:\Documents\Code\gitlab-automation-suite\app\main\services.py:27]
2025-08-14 15:22:59,297 INFO: Successfully connected to GitLab at https://emu.hinagro.com [in D:\Documents\Code\gitlab-automation-suite\app\main\services.py:27]
2025-08-14 15:23:09,836 INFO: Successfully connected to GitLab at https://emu.hinagro.com [in D:\Documents\Code\gitlab-automation-suite\app\main\services.py:27]
2025-08-14 15:23:09,837 INFO: Generating Issue TAT Trend Report for 3 months. [in D:\Documents\Code\gitlab-automation-suite\app\main\logic.py:211]
2025-08-14 15:23:17,308 INFO: Fetched ALL 385 issues with filters: {'state': 'closed', 'created_after': '2025-05-14', 'created_before': '2025-08-14', 'all': True, 'include_subgroups': True} [in D:\Documents\Code\gitlab-automation-suite\app\main\services.py:279]
2025-08-14 15:24:54,159 INFO: Successfully connected to GitLab at https://emu.hinagro.com [in D:\Documents\Code\gitlab-automation-suite\app\main\services.py:27]
2025-08-14 15:24:54,163 INFO: Generating Issue TAT Trend Report for 12 months. [in D:\Documents\Code\gitlab-automation-suite\app\main\logic.py:211]
2025-08-14 15:25:21,727 INFO: Fetched ALL 1456 issues with filters: {'state': 'closed', 'created_after': '2024-08-14', 'created_before': '2025-08-14', 'all': True, 'include_subgroups': True} [in D:\Documents\Code\gitlab-automation-suite\app\main\services.py:279]
2025-08-18 10:29:30,599 ERROR: GitLab connection failed: HTTPSConnectionPool(host='emu.hinagro.com', port=443): Max retries exceeded with url: /api/v4/user (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x000001E09F3C3D90>, 'Connection to emu.hinagro.com timed out. (connect timeout=10)')) [in D:\Documents\Code\gitlab-automation-suite\app\main\services.py:29]
2025-08-18 10:30:13,187 INFO: Successfully connected to GitLab at https://emu.hinagro.com [in D:\Documents\Code\gitlab-automation-suite\app\main\services.py:27]
